### PR TITLE
PIC-4008 add prometheus alerting on SQS

### PR DESCRIPTION
**Pull request by markreesmoj**
This was significantly complicated by the fact that this service is on helm v1, and does not already have alerting or the generic service set up.

### CODE REVIEW

File: helm_deploy/court-case-matcher/templates/requirements.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+metadata:
```
Comment (markreesmoj) [inline helm_deploy/court-case-matcher/templates/requirements.yaml:1]: This is the only way I could get the linter to pass

File: helm_deploy/values-dev.yaml [hunk 2]
```diff
@@ -32,3 +32,12 @@ resources:
   memory:
     limit: 1200Mi
     request: 350Mi
+
+generic-prometheus-alerts:
+  targetApplication: court-case-matcher
+  alertSeverity: slack-hmpps-user-preferences
```
Comment (markreesmoj) [inline helm_deploy/values-dev.yaml:38]: Will send messages to `probation_in_court_alerts`
I wanted to get something in place ASAP - would be better to go with setting up webhooks for dev/preprod/prod but that would be much more work

File: helm_deploy/court-case-matcher/templates/requirements.yaml (hunk 1 above)
Comment (markreesmoj) [reply helm_deploy/court-case-matcher/templates/requirements.yaml:1]: It's a warning on my machine but fails in the pipeline, presumably different versions of helm

File: helm_deploy/values-dev.yaml (hunk 2 above)
Comment (Obsiye) [reply helm_deploy/values-dev.yaml:38]: Hi, we have webhooks setup already  to go to the specific slack channels

1. severity = "probation_in_court_alerts_dev"
2. severity = "probation_in_court_alerts_preprod"
3. severity = "probation_in_court_alerts_prod"

Comment (Obsiye) [reply helm_deploy/values-dev.yaml:38]: Hi, we have webhooks setup already  to go to the specific slack channels

1. severity = "probation_in_court_alerts_dev"
2. severity = "probation_in_court_alerts_preprod"
3. severity = "probation_in_court_alerts_prod"

**Review by Obsiye — DISMISSED** [review]

**Review by Obsiye — APPROVED** [review]

**Pull request merged**